configure.ac

```diff
@@ -37,14 +37,23 @@
 #
 
 AC_PREREQ([2.63])
-AC_INIT([pbspro], [18.1.0], [pbssupport@altair.com])
+# Use PBS_VERSION to override the version statically defined here. For example:
+# ./configure PBS_VERSION=19.1.2 --prefix=/opt/pbs
+AC_INIT([PBS Professional],
+  [19.0.0],
+  [pbssupport@altair.com],
+  [pbspro],
+  [http://www.pbspro.org/])
 AC_CONFIG_HEADERS([src/include/pbs_config.h])
 AC_CONFIG_SRCDIR([src/cmds/qmgr.c])
 AC_CONFIG_AUX_DIR([buildutils])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CANONICAL_TARGET([])
-# FIXME: The subdir-objects option will be enabled by default in the future
-AM_INIT_AUTOMAKE([-Wall foreign])
+os_id=`grep ^ID= /etc/os-release | sed -n 's/.*"\(.*\)"/\1/p'`
+AS_CASE([$os_id],
+  [opensuse-tumbleweed], m4_define([am_init_string], [-Wall foreign subdir-objects]),
```

Comment:
Here's a sample the output I see on Tumbleweed without it:

src/lib/Libpbs/Makefile.am:55: warning: source file '../Libattr/attr_fn_arst.c' is in a subdirectory,
src/lib/Libpbs/Makefile.am:55: but option 'subdir-objects' is disabled
automake: warning: possible forward-incompatibility.
automake: At least a source file is in a subdirectory, but the 'subdir-objects'
automake: automake option hasn't been enabled.  For now, the corresponding output
automake: object file(s) will be placed in the top-level directory.  However,
automake: this behaviour will change in future Automake versions: they will
automake: unconditionally cause object files to be placed in the same subdirectory
automake: of the corresponding sources.
automake: You are advised to start using 'subdir-objects' option throughout your
automake: project, to avoid future incompatibilities.

There are MANY of these warnings, so it's basically taking automake's advice. The other platforms don't complain, and I didn't want to risk compatibility issues with how we currently build.